app/src/main/res/layout/topic_index_items.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:orientation="vertical"
+  android:id="@+id/cardView"
+  android:layout_width="match_parent"
+  android:layout_height="@dimen/card_view_height"
+  android:layout_margin="@dimen/margin_5dp"
+  app:cardCornerRadius="@dimen/corner_radius_3dp">
+
+  <LinearLayout
+    android:id="@+id/llContainer"
+    android:background="@color/lightBrown"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ImageView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_margin="@dimen/margin_8dp"
+      android:src="@mipmap/ic_launcher"/>
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/brown"
+      android:layout_marginTop="@dimen/margin_8dp"
+      android:padding="@dimen/padding_8dp"
+      android:orientation="vertical">
+
+      <TextView
+        android:id="@+id/tvTitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textColor="@color/white"
+        android:textStyle="bold"
+        android:textSize="@dimen/text_size18sp"
+        android:text=""/>
+
+      <TextView
+        android:id="@+id/tvCategory"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textColor="@color/white"
+        android:layout_marginTop="@dimen/margin_5dp"
+        android:text=""/>
+
+      <RelativeLayout
+        android:id="@+id/llSubContainer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <TextView
+          android:id="@+id/tvDescription"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:maxLines="3"
```

Comment:
Can we define this number `3` in xml file?

Reply:
yes it can be done.